### PR TITLE
Fix landscape navigationbar orientation in designer

### LINKED ISSUE #1201: Navigationbar in landscape mode is on wrong side
On every android device is the navigationbar in landscpae mode on the right side.

On the app inventor phone preview is it on the bottom side.
01 - kopie

### PR DESCRIPTION
**Pull request by bartmathijssen**
I am new to GWT so I would like some feedback on the changes I made. I feel like the `NavigationBar` widget is almost obsolete at this point, but I can't get this functionality to work without it. 

Resolves #1201

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

**Comment by ewpatton** [comment]
@bartmathijssen Yes there is some legacy stuff in there that could probably be removed. We'll be starting on a revamp of the UI including using some newer GWT technologies next year and will probably use that as an opportunity to review and remove unneeded code.

File: appinventor/appengine/war/Ya.css [hunk 1]
```diff
@@ -988,6 +988,19 @@ select.ode-PropertyEditor[disabled] {
   height: auto !important;
   background-color: #d2e0a6;
   border: 2px solid black;
+  display: flex;
```
Comment (ewpatton) [inline appinventor/appengine/war/Ya.css]: Also include -webkit prefix version to support older versions of Safari

File: appinventor/appengine/war/Ya.css [hunk 2]
```diff
@@ -988,6 +988,19 @@ select.ode-PropertyEditor[disabled] {
...
+}
+
+.ode-SimpleMockFormPhoneLandscape .ode-SimpleMockForm,
+.ode-SimpleMockFormPhoneLandscapeTablet .ode-SimpleMockForm,
+.ode-SimpleMockFormPhoneLandscapeMonitor .ode-SimpleMockForm {
+  flex-direction: row;
```
Comment (ewpatton) [inline appinventor/appengine/war/Ya.css]: Add -webkit version

File: appinventor/appengine/war/Ya.css [hunk 3]
```diff
@@ -988,6 +988,19 @@ select.ode-PropertyEditor[disabled] {
...
+}
+
+.ode-SimpleMockFormPhonePortrait .ode-SimpleMockForm,
+.ode-SimpleMockFormPhonePortraitTablet .ode-SimpleMockForm,
+.ode-SimpleMockFormPhonePortraitMonitor .ode-SimpleMockForm {
+  flex-direction: column;
```
Comment (ewpatton) [inline appinventor/appengine/war/Ya.css]: Add -webkit version

**Comment by bartmathijssen** [comment]
@ewpatton I updated the supported Safari version and added the -webkit prefixes to the css properties.

**Pull request merged**